test-it/ExpectedResults/ComplexMathFunctions.vtr

```diff
@@ -40,3 +40,5 @@ Log (0 99) => (4.59512 1.5708)
 Log2 (0 99) => (6.62936 2.26618)
 Log10 (0 99) => (1.99564 0.682188)
 Log10 ((1.75517 0.958851) (1.62091 2.52441)) => ((0.30103 0.217147) (0.477121 0.434294))
+Absolute Double (-96.7324 -265.621) => 282.6865193998785912
```

Comment:
@cglzaguilar How far off is the browser calculation using stdlib complex abs() directly?
This is a pretty straight-forward calculation on numbers with nearly the same exponent and mantissa range.  The expected value in the test is the same answer CG/NCG computes (using both methods).  While it's always a good idea to avoid exact comparison of floating point values, I wouldn't expect the answer to be off my more than one dbl machine epsilon (2.2e-16).
So I would say, if the answer is within +/- one epsilon, we might consider rolling this back and fixing the test only.  Otherwise this fix is necessary (but the test should still probably be fixed to do an epsilon comparison).